prime-router/src/main/kotlin/Metadata.kt

```diff
@@ -46,7 +46,7 @@ class Metadata : Logging {
         NullMapper(),
         NullDateValidator()
     )
-    private var jurisdictionalFilters = listOf(
+    private var reportStreamFilterDefinitions = listOf(
```

Comment:
Throughout, the concept of filters has been generalized.   The new general name is "ReportStreamFilters"